### DISCUSSION TITLE
Self-hosted instance always makes API requests over IPv4 even when IPv6 is available

### DISCUSSION THREAD

**Discussion by dumbmoron** [plausible/analytics#3915] [Self-Hosted Support]
The self-hosted instance keeps trying to use IPv4 for all requests it makes (e.g. loading data from Google Analytics, avatars) despite there being AAAA (IPv6 DNS) records available for these services. This makes it impossible to use these parts of the service on an IPv6-only machine.

The result is usually a `nxdomain` error in the logs or a timeout.

Would it be possible to at least add some environment variable to enable IPv6 when making requests? I noticed that Mint [had an option added for it](https://github.com/elixir-mint/mint/blob/main/CHANGELOG.md#v120) in v1.2.0.

**Comment by ruslandoga** [comment]
👋 @dumbmoron 

There was a PR that added `FINCH_IPV6` but it was closed https://github.com/plausible/analytics/pull/2737
